guides/source/association_basics.md

```diff
@@ -2166,18 +2305,6 @@ class Hardback < ApplicationRecord; end
 class Paperback < ApplicationRecord; end
 ```
 
-##### `:strict_loading`
-
-When set to true, enforces strict loading every time the associated record is loaded through this association.
-
-##### `:through`
-
-The `:through` option specifies a join model through which to perform the query. `has_many :through` associations provide a way to implement many-to-many relationships, as discussed [earlier in this guide](#the-has-many-through-association).
-
-##### `:validate`
-
-If you set the `:validate` option to `false`, then new associated objects will not be validated whenever you save this object. By default, this is `true`: new associated objects will be validated when this object is saved.
-
 #### Scopes for `has_many`
```

Comment:
All these "Scopes" secionts for each association have a lot of duplication.
I think it would make sense to have a single top section ("Scopes for associations", "Customizing Association Scope") and remove the duplication.